wolfssl/openssl/sha.h

```diff
@@ -18,7 +39,7 @@
 
 typedef struct WOLFSSL_SHA_CTX {
     /* big enough to hold wolfcrypt Sha, but check on init */
-    int holder[28 + (WC_ASYNC_DEV_SIZE / sizeof(int))];
+    void* holder[(112 + WC_ASYNC_DEV_SIZE) / sizeof(void*)];
```

Comment:
Why have we gone from 28 to 112?  That's a huge increase.  Same for following items.

Reply:
The reason for the change in constant values is that all were converted from int (or long long) to size in bytes. Having the numerator be the total number of bytes and then dividing it by the sizeof(void*).